### PR TITLE
Add python setuptools in the required packages. 

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Add python setuptools in the required packages. 

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/723453742
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/723342203

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on signing off your commits, please check [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/CONTRIBUTING.md#sign-your-work).

### CODE REVIEW

File: .github/workflows/prod-sync-artifacts.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ name: Sync ODFE artifacts to Prod
 on: 
   repository_dispatch:
     types: [prod-sync-artifacts]
+  push:
+    branches: [sreekarj_prodsync]
```
Comment (peterzhuamazon) [inline .github/workflows/prod-sync-artifacts.yml:11]: Remove these

File: .github/workflows/prod-sync-artifacts.yml [hunk 2]
```diff
@@ -24,6 +26,7 @@ jobs:
         run: ./release-tools/scripts/required_packages.sh
       - name: Install python packages
         run: |
+          sudo apt-get install python-setuptools
```
Comment (peterzhuamazon) [inline .github/workflows/prod-sync-artifacts.yml:29]: This should be included in the required_packages.sh since it is ubuntu specific.

File: release-tools/scripts/manifest.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ urls:
     prod: s3://artifacts.opendistroforelasticsearch.amazon.com/
     #snapshots: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/
     releases: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/releases/
-    releases_final_build: 
+    releases_final_build: rc-build-712915909
```
Comment (peterzhuamazon) [inline release-tools/scripts/manifest.yml:31]: This should be in its separate PR.

File: .github/workflows/prod-sync-artifacts.yml (hunk 1 above)
Comment (sreekarjami) [reply .github/workflows/prod-sync-artifacts.yml]: This is removed.

File: release-tools/scripts/manifest.yml (hunk 3 above)
Comment (sreekarjami) [reply release-tools/scripts/manifest.yml]: Opening a separate PR.

File: .github/workflows/prod-sync-artifacts.yml (hunk 2 above)
Comment (sreekarjami) [reply .github/workflows/prod-sync-artifacts.yml]: Added in the required packages.

**Review by gaiksaya — APPROVED** [review]

File: release-tools/scripts/required_packages.sh [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 ARCH=`uname -p | tr '[:upper:]' '[:lower:]'`; echo $ARCH
 PLATFORM=`uname -s | tr '[:upper:]' '[:lower:]'`; echo $PLATFORM
-DEB_PKGS="curl wget unzip tar jq python python3 git awscli libnss3-dev fonts-liberation libfontconfig1"
+DEB_PKGS="curl wget unzip tar jq python python3 git awscli libnss3-dev fonts-liberation libfontconfig1 python-setuptools"
 RPM_PKGS="curl wget unzip tar jq python python3 git awscli libnss3.so xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi \
-          xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype"
+          xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype python-setuptools"
```
Comment (peterzhuamazon) [inline release-tools/scripts/required_packages.sh:7]: Did you check if yum actually has python-setuptools?

Comment (sreekarjami) [reply release-tools/scripts/required_packages.sh:7]: Yes. yum has python-setuptools

**Review by camerski — APPROVED** [review]
